### ISSUE TITLE
feature: use system package for thirdparty software when available

### ISSUE THREAD

**Issue by Trophime** [OpenModelica/OpenModelicaBuildScripts#24]
Hi,
would it be possible to use system package for thirdparty software when available?
At least on Debian/Ubuntu 

Best

**Comment by sjoelund** [comment]
No. It would be a nightmare to test it properly. Many pieces of software is not fully source-compatible either (like Sundials). It is also often compiled with the wrong flags (like skipping -fPIC when creating static libraries).
